packages/wallets/src/components/Base/Divider/Divider.tsx

```diff
@@ -8,7 +9,10 @@ type TProps = {
 };
 
 const Divider: React.FC<TProps> = ({ color = '#f2f3f4', margin, variant = 'horizontal' }) => (
-    <div className={`wallets-divider__${variant}`} style={{ borderColor: color, margin }} />
+    <div
+        className={classNames('wallets-divider', `wallets-divider__${variant}`)}
```

Comment:
```suggestion
        className={`wallets-divider wallets-divider__${variant}`}
```

You don't need classNames here in my opinion

Reply:
Ohh yea 👀